sources/AppBundle/Email/Emails.php

```diff
@@ -37,10 +37,55 @@ public function sendInscription(Event $event, MailUser $recipient)
             $message->addAttachment(new Attachment(Event::getInscriptionAttachmentFilepath($eventPath), $event->getTitle() . '.pdf', 'base64', 'application/pdf'));
         }
 
+        $message->addAttachment($this->getAttachementIcsInscription($event, $recipient));
+
         $this->mailer->renderTemplate($message, ':admin/event:mail_inscription.html.twig', [
+            'event' => $event,
+            'recipient' => $recipient,
             'content' => $mailContent,
             'logo_url' => $event->getLogoUrl(),
         ]);
         $this->mailer->send($message);
     }
+
+    private function getAttachementIcsInscription(Event $event, MailUser $recipient)
+    {
+        $uid = md5($event->getId());
+        $organizerCN = MailUserFactory::afup()->getName();
+        $organizerEmail = MailUserFactory::afup()->getEmail();
+        $attendeeCN = $recipient->getName();
+        $attendeeEmail = $recipient->getEmail();
+        $created = (new \DateTime())->setTimezone(new \DateTimeZone('UTC'))->format('Ymd\THis\Z');
+
+        $content = <<<EOF
+BEGIN:VCALENDAR
+PRODID:-//AFUP//AFUP 0.0.7//FR
+VERSION:2.0
+CALSCALE:GREGORIAN
+METHOD:REQUEST
+BEGIN:VEVENT
+DTSTART:{$event->getDateStart()->setTimezone(new \DateTimeZone('UTC'))->format('Ymd\T000000\Z')}
```

Comment:
quand je checkoute la branche en local et fait un test d'inscription, puis récupère le fichier ics et me l'envoie.

çà donne cela :

![Capture d’écran du 2022-12-04 19-09-48](https://user-images.githubusercontent.com/320372/205507977-f66b24b1-0df8-4dfa-bff1-25bc245b81eb.png)

où dans le calendrier, l'événement n'est pas sur toute la journée, mais sur une heure précisée, et à 2h la veille. 

![Capture d’écran du 2022-12-04 19-11-14](https://user-images.githubusercontent.com/320372/205507984-82463567-cbf9-41cd-bd2f-981d78b7c1c3.png)

Tu saurais pourquoi ?

Reply:
Le problème venait du fait que je forçais les heures, minutes, secondes à 00.
C'est bon maintenant.